force-app/main/default/classes/OSF_ContractTriggerTest.cls

```diff
@@ -0,0 +1,58 @@
+@isTest
```

Comment:
try to add at least 1 failure scenario (when you catch an error or smth like that)